README.spanish.md

```diff
@@ -499,7 +526,7 @@ Todo lo de encima hubiera devuelto `false` si se hubiera usado `===`
 
 **TL;DR:** La prueba de extremo a extremo (e2e) que incluye datos en vivo solía ser el eslabón más débil del proceso de CI, ya que depende de múltiples servicios pesados como DB. Usa un entorno que esté lo más cerca posible de su producción real como a-continue
 
-**De lo contrario:** Sin los equipos compuestos por docker, deben mantener una base de datos de prueba para cada entorno de prueba, incluidas las máquinas de los desarrolladores, mantener todas esas bases de datos sincronizadas para que los resultados de la prueba no varíen entre entornos.
+**De lo contrario:** Sin 'docker-compose',los equipos deben mantener una base de datos de prueba para cada entorno de prueba, incluidas las máquinas de los desarrolladores, mantener todas esas bases de datos sincronizadas para que los resultados de la prueba no varíen entre entornos.
```

Comment:
Not sure about it since i don't know docker, but it seems like `docker-compose` is like a tool or something.

The one above says "without the teams composed by docker". Which misses the `,` of the english version

Reply:
> Not sure about it since i don't know docker, but it seems like `docker-compose` is like a tool or something.
> 
> 
> 
> The one above says "without the teams composed by docker". Which misses the `,` of the english version
> 
> 

Yes, `docker-compose` is a tool